### PR TITLE
added instructions on how to get ocaml

### PR DESCRIPTION
**Pull request by UnixJunkie**
once you have opam

### CODE REVIEW

**Comment by UnixJunkie** [comment]
Recently, people were complaining on caml-list about the quality of the documentation.
Let's hope this makes things clearer.
Feel free to edit to whatever you think would be better.

**Comment by UnixJunkie** [comment]
On purpose, I did not install the latest version of the compiler.
Because, if you install the latest version, sometimes some packages are broken.
So, I feel safer to point users to a version which is not the bleeding edge, so that
there are more chances they can install what they are looking for afterwards.

File: site/docs/install.md [hunk 1]
```diff
@@ -29,6 +29,20 @@ the recommended way to install the OCaml compiler and OCaml
 packages. Simply follow the
 [OPAM install instructions](https://opam.ocaml.org/doc/Install.html).
 
+Then, use opam to install an ocaml compiler.
+Here are the commands for a Linux user using the Bash shell and opam-2.0.0:
```
Comment (dbuenzli) [inline site/docs/install.md]: I don't think that information is relevant. I'd remove it.

File: site/docs/install.md [hunk 2]
```diff
@@ -29,6 +29,20 @@ the recommended way to install the OCaml compiler and OCaml
...
+```
+# environment setup
+opam init
+eval `opam env`
+# install given version of the compiler
+opam switch create 4.06.1
```
Comment (dbuenzli) [inline site/docs/install.md:39]: Why not use the latest version here ?

Comment (dbuenzli) [reply site/docs/install.md:39]: Sorry didn't read your message properly. Makes sense.

File: site/docs/install.md (hunk 1 above)
Comment (UnixJunkie) [reply site/docs/install.md]: I've updated it but not removed it.
I think eval is a bash thing.
Again, feel free to edit this PR.

**Comment by pw374** [comment]
Thanks!

**Pull request merged**